java/src/com/google/idea/blaze/java/run/fastbuild/FastBuildTestEnvironmentCreator.java

```diff
@@ -140,10 +142,27 @@ GeneralCommandLine createCommandLine(
     addJvmOptsFromBlazeFlags(config, commandBuilder);
 
     for (String flag : targetJavaInfo.jvmFlags()) {
+      // Wix tooling injects some java arguments which makes java call fail
+      // Temporary filtering them out to see if it's makes run succeed
+      // ex: "-javaagent:$(rootpath @core_server_build_tools//test-agent/src/main/java/com/wixpress/agent:test-agent_deploy.jar)"
+      if (flag.contains("@core_server_build_tools")) {
```

Comment:
this is actually another hack which we should track. It's related though to IT running so maybe that's enough